plugins/package-managers/conan/src/funTest/assets/projects/synthetic/conan-expected-output-py-lockfile.yml

```diff
@@ -0,0 +1,336 @@
+---
```

Comment:
Previously, the expected results have been named after the synthetic project.

Should we also create a dedicated project for the lockfile case for clarity?